src/Courses/CourseDetailsTabs/StatisticsTab.tsx

```diff
@@ -137,6 +138,7 @@ export const StatisticsTab: React.FC<StatisticsTabProps> = ({ course, userId })
     const globalView = statisticsViewFromAllStatisticsViewFilter(view);
 
     useEffect(() => {
+        logger.silly(`Stats tab: [useEffect] course.id ${course.id}, globalView ${globalView}, idFilter ${idFilter}, userId ${userId}, userType ${userType}`);
```

Comment:
We need some cut and dry reasoning for using silly
Right now the only issue is that console does not have a silly log so it will log with console.log

Reply:
okay, so info then? This was just an attempt to follow the render cycle for this component

Reply:
@red-tt-bear if we're going to balk at using custom log-levels because they all log as Log, then we probably should remove them and stick to the browser defaults.

I think the reasoning for silly is straightforward- it's a log that is verbose or unlikely to be helpful.